### PR TITLE
linux install

### PR DESCRIPTION
**Pull request by rosemarymarano**
First cut at adding information for a Linux install.

### CODE REVIEW

**Comment by jshum2479** [comment]
Hi @rosemarymarano , for ubuntu :

1.  Download the latest wktui deb package from https://github.com/oracle/weblogic-toolkit-ui

2.  Run `sudo dpkg -i wktui_1.0.0_amd64.deb`,  you may or may not see below

```
Selecting previously unselected package wktui.
(Reading database ... 70158 files and directories currently installed.)
Preparing to unpack wktui_1.0.0_amd64.deb ...
Unpacking wktui (1.0.0-784) ...
dpkg: dependency problems prevent configuration of wktui:
 wktui depends on gconf2; however:
  Package gconf2 is not installed.
 wktui depends on gconf-service; however:
  Package gconf-service is not installed.
 wktui depends on libnotify4; however:
  Package libnotify4 is not installed.
 wktui depends on libappindicator1; however:
  Package libappindicator1 is not installed.
 wktui depends on libxtst6; however:
  Package libxtst6 is not installed.
 wktui depends on libsecret-1-dev; however:
  Package libsecret-1-dev is not installed.

dpkg: error processing package wktui (--install):
 dependency problems - leaving unconfigured
Processing triggers for shared-mime-info (1.15-1) ...
Processing triggers for mime-support (3.64ubuntu1) ...
Errors were encountered while processing:
 wktui
```

3.  If you see above, then run

```
sudo apt-get install -f
```

3. If you run wktui from os terminal shell

```
wktui
```

You may see this

```
wktui
libGL error: No matching fbConfigs or visuals found
libGL error: failed to load driver: swrast```

This is harmless and can be removed by setting the environment variable before launching wktui

```
export LIBGL_ALWAYS_INDIRECT=1
```

File: documentation/1.0/content/setup/install.md [hunk 1]
```diff
@@ -9,6 +9,60 @@ description: "Install the WKT UI application and check for updates."
 1. Download the latest WebLogic Kubernetes Toolkit UI (WKT UI) application installers from the [GitHub Releases section](https://github.com/oracle/weblogic-toolkit-ui/releases) of this repository.
 2. Simply run the appropriate installer for your operating system.
 
+**NOTE**: When installing on Linux, to get _all_ the dependencies and installed in the correct order, use a command similar to the following:
+```
+sudo yum localinstall wktui-1.0.0.x86_64.rpm
+```
+
+- On Oracle/RedHat/CentOS (and some others), use either `yum` or `dnf`
```
Comment (robertpatrick) [inline documentation/1.0/content/setup/install.md:17]: I would move this up above the `sudo yum ...` line since yum doesn't exist on Ubuntu...

Comment (rosemarymarano) [reply documentation/1.0/content/setup/install.md:17]: ok, got it.

File: documentation/1.0/content/setup/install.md [hunk 2]
```diff
@@ -9,13 +9,14 @@ description: "Install the WKT UI application and check for updates."
...
+**NOTE**: On Linux, to get _all_ the dependencies and have them installed in the correct order, you need to use the package manager to install the `rpm` or `deb` file.
+
+- On Oracle/RedHat/CentOS (and some others), use either `yum` or `dnf`; for example:
 ```
 sudo yum localinstall wktui-1.0.0.x86_64.rpm
 ```
 
-- On Oracle/RedHat/CentOS (and some others), use either `yum` or `dnf`
-- On Ubuntu, run:
+- On Ubuntu, use `apt-get`; for example:
```
Comment (rosemarymarano) [inline documentation/1.0/content/setup/install.md:19]: @robertpatrick @jshum2479 I know that there might still be some discussion going on here about this, but I wonder if we need another sentence/explanation between stating, "On Ubuntu, use `apt-get`; for example:" and "sudo dpkg -i wktui_1.0.0_amd64.deb," something to explain why we are showing this command `sudo dpkg -i wktui_1.0.0_amd64.deb`.

Comment (robertpatrick) [reply documentation/1.0/content/setup/install.md:19]: @jshum2479 @rosemarymarano It's not clear to me that we need to show the usage of `dpkg -i`.  Unless there is some problem that the `dpkg -i` is solving that prevents the `apt-get` command from working, I would opt to remove the `dpkg -i` section completely and just show the `apt-get` command.  

If there is a problem it is trying to solve, then I agree that we need an explanation of the problem, what in the `dpkg -i` output the user should look for, and how the follow-on `apt-get` command solves the problem.  However, we probably need the basic `apt-get` command prior to discussing a potential problem that might be encountered and how to address it.  Can the two of you please work together to find the right organization of this section?

**Review by jshum2479 — CHANGES_REQUESTED** [review]
I have some comments but it's in pending status

**Review by robertpatrick — APPROVED** [review]

**Pull request merged**